package.json

```diff
@@ -41,7 +41,7 @@
     "@rails/webpacker": "5.4.3",
     "babel-loader": "^8.2.3",
     "cable_ready": "5.0.0-pre9",
-    "stimulus_reflex": "3.5.0-pre9",
+    "debounced": "^0.0.5",
```

Comment:
Two changes in one commit. This creates conflicts whenever stimulus_reflex is updated while this branch lives. It's tiny but a separate PR moving the stimulus line can be merged much quicker and avoids extra work of resolving conflicts. And maybe there's more to fix?

Reply:
Yep, I've opened a PR: https://github.com/openfoodfoundation/openfoodnetwork/pull/9865
Once it's green, I merge it, and then I rebase this one ;)

Reply:
Done!